src/CommandInvocationParser.cpp

```diff
@@ -18,6 +18,75 @@ namespace cmake::language
     {
       return { std::find_if_not(r.begin, r.end, [](char c) {return std::isspace(c); }), r.end };
     }
+
+    //-----------------------------------------------------------------------------
+    tl::expected<Token, Error> ParserIdentifier(Range r)
+    {
+      auto newRange = IgnoreSpaces(r);
+      auto identifier = Parser<ElementType::Identifier>{}.Parse(newRange);
+      if (identifier)
+      {
+        // Make sure we take into account the spaces
+        identifier->range.begin = r.begin;
+      }
+      return identifier;
+    }
+
+    //-----------------------------------------------------------------------------
+    tl::expected<Token, Error> ParserArguments(Range r)
+    {
+      auto newRange = IgnoreSpaces(r);
+      if (*newRange.begin != '(')
+      {
+        Error err;
+        err.message = "A command invocation should wrap its arguments between parenthesis.";
+        err.context = newRange.begin;
+        return tl::make_unexpected(err);
+      }
+
+      auto args = Parser<ElementType::Arguments>{}.Parse(Range{ newRange.begin + 1, r.end });
+      if (args)
+      {
+        if (args->range.end >= r.end)
+        {
+          Error err;
+          err.message = "Missing parenthesis";
+          err.context = args->range.end;
+          return tl::make_unexpected(err);
+        }
+
+        if (*args->range.end != ')')
+        {
+          Error err;
+          err.message = fmt::format("Expected \')\', got {}", *args->range.end);
```

Comment:
We already checked that the opening parenthesis is `(` so the error should be "got `}`" IMO.

Reply:
Here, the `{}` are just placeholders that will be replaced by the value of `*args->range.end`.